tasks/e2e-installs.sh

```diff
@@ -146,6 +146,20 @@ grep '"version": "0.4.0"' node_modules/react-scripts/package.json
 checkDependencies
 
 # ******************************************************************************
+# Test --use-npm flag
+# ******************************************************************************
+
+cd "$temp_app_path"
+create_react_app --use-npm --scripts-version=0.4.0 test-use-npm-flag
```

Comment:
I don't think we need to install a specific version here.

Reply:
Shouldn't hurt 😄